docs/Enterprise.md

```diff
@@ -0,0 +1,26 @@
+# Configuring Zulip Desktop for multiple users
+
+If you're a system admin and want to add certain organizations to the Zulip app for
+all users of your system, you can do so by creating an enterprise config file.
+The file should be placed at `/etc/zulip-desktop-config` for Linux and macOS computers
+and inside `C:\Program Files\Zulip-Desktop-Config` on Windows.
+It must be named `enterprise_config.json` in both cases. 
+
+To specify the preset organization you want to add for other users, you will need to
```

Comment:
Change to "organizations".

Reply:
Considering the flow of the docs and the example below this should be left as `organization` because we give a example of specifying a single organization.